confgenerator/testdata/valid/windows/logging-receiver_files_refresh_interval/input.yaml

```diff
@@ -16,11 +16,13 @@ logging:
   receivers:
     log_source_id1:
       type: files
-      wildcard_refresh_interval: 10s
       include_paths:
-      - /path/to/log/1/a
+      - /path/to/log/1/*
```

Comment:
Might be worth splitting this out into a separate test, or at least leaving a comment that this file is testing extra functionality.

Reply:
The old test was invalid, because it was purporting to test the wildcard refresh interval configuration without a wildcard.